omc3/model/accelerators/lhc.py

```diff
@@ -384,6 +388,19 @@ def get_update_correction_script(self, outpath: Path, corr_file: Path) -> str:
         )
         return madx_script
 
+    def _uses_ats_knobs(self) -> bool:
+        """Returns wether the ATS knobs and macros should be used, based on the instance's year."""
+        try:
+            return int(self.year) >= 2018  # self.year is always a string
```

Comment:
and here there should be a `==`, since using ATS knobs is an outlier from OP's point of view

Reply:
that's what I feared. Easy fix :). Thx!